docs/asyncapi.yaml

```diff
@@ -222,7 +222,7 @@ components:
                   properties:
                     filename:
                       $ref: '#/components/schemas/Filename'
-    PaymentRequestAHWR:
+    PaymentRequest:
       type: object
       description: Vet Visit payment object
```

Comment:
Would this description need to change?

Reply:
Yes it should, updated.

Reply:
I'll take a look to see if there are any more references to vet visits in the doc